exareme2/worker/worker_info/sqlite.py

```diff
@@ -0,0 +1,16 @@
+import sqlite3
```

Comment:
I think we should keep the same connection open, using a factory like method.

We are going to have only write operations so there is no need for locks etc.

Reply:
You mean read operations